scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/ForkedTaskVariablesManager.java

```diff
@@ -125,6 +132,17 @@ public SchedulerNodeClient createSchedulerNodeClient(TaskContext container) {
         return null;
     }
 
+    public RMNodeClient createRMNodeClient(TaskContext container) {
```

Comment:
It would be good to document the case where this method return null value.

Reply:
ok